content/1_docs/2_cookbook/9_setup/0_kirby-in-a-nutshell/recipe.txt

```diff
@@ -0,0 +1,318 @@
+
+This is no problem as long as you stick with your field types. However, if you decide to make changes to field types in a blueprint during development, or you allow changing templates, you have to be a bit careful.
+
+<info>
+- Make sure that the templates that a user can select from when changing templates are compatible.
+- Keep in mind that content is deleted if the field exists in the target template but has a different field type.
+</info>
+
+## Sections or fields?
+
+Pages and files can be organized in the Panel using files/pages fields or files/pages sections. It is important to understand the difference between these two types to make informed decisions when to use what.
+
+### Pages section vs. pages field
+
+(screenshot: pages-section-vs-pages-field.png)
+
+#### Pages section
+
+- Shows a list of existing pages with an option to create new pages.
+- Pages in a pages section can only have a single parent page which is defined with the `parent` option.
+- Pages are filterable by their template + status only.
+- A pages section doesn't store anything in the content file.
+
+#### Pages field
+
+- Is basically a select field that allows selecting pages from a given set of pages.
+- The selectable pages can be from different parents, the set is defined via the `query` option. If no query is defined, the field options default to the children of the current page.
+- The ID(s) of the selected page(s) are stored in the content file.
+- Currently there is no option to create new pages from a pages field.
+- A typical use case is creating references between pages, for example to manually select related pages.
+
+### Files section vs. files field
+
+(screenshot: files-section-vs-files-field.png)
+
+#### Files field
+
+- Select one or more files from a (larger) set of files
+- The ID(s) of the selected file(s) are stored in the page’s content file
+- Upload a single file at a time, the newly uploaded file is then automatically selected
+- Deleting a file uploaded via the files field is only possible via the file view (or through a files section), not directly from the files field
+- Templates can be assigned to uploaded files via the upload option
+- The files to select from can be defined using the `query` option. The `query` allows you to fetch files from different parents and to filter the set by any criteria. If no query is set, the field defaults to files of the current page
+- You can only upload to a single page. The default parent is the current page
+
+#### Files section
+
+- Upload one or more files at once to a single parent
+- Files can only be uploaded to a single page. The destination is set with the `parent` option
+- Files can be filtered by template only
+- You can limit what type of files can be uploaded with the `accept` option in the given files template
+- Files are uploaded to the given parent
+- The IDs of the uploaded files are not stored anywhere
```

Comment:
I'd swap the explanation of the field and section around to mirror the order of the pages field and section. Otherwise it could be confusing if the reader wants to compare the pages to the files field for example.

Reply:
✔︎